user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_behavioral_anomaly_detection.md

```diff
@@ -31,7 +31,7 @@ From DataMiner 10.0.0/10.0.2 onwards, the DataMiner Analytics software can detec
 
 - **Unlabeled change**: If a change point cannot be classified as one of the above-mentioned change points, it is considered an unlabeled change.
 
-If a change point other than an outlier or unlabeled change is unexpected, it will be considered anomalous. Level shifts that have a different direction than previous recent jumps or that jump to a previously unseen level will typically be labeled “anomalous”. Similarly, trend or variance changes will be labeled “anomalous” when no earlier trend or variance changes in the same direction were detected during the last weeks. A flatline will be considered anomalous when no recent flatline change point of approximately the same length or longer is detected.
+If a change point other than an unlabeled change is unexpected, it will be considered anomalous. Prior to DataMiner 10.3.12/10.4.0<!--RN 37171-->, if a change point other than an outlier or unlabeled change is unexpected, it will be considered anomalous. Level shifts that have a different direction than previous recent jumps or that jump to a previously unseen level will typically be labeled “anomalous”. Similarly, trend or variance changes will be labeled “anomalous” when no earlier trend or variance changes in the same direction were detected during the last weeks. A flatline will be considered anomalous when no recent flatline change point of approximately the same length or longer is detected.
```

Comment:
Don't know how detailed we want to go here, but an outlier will only be considered an anomaly if alarming for outliers is turned on (at least for now, this might change in the future cfr. DCP223402). I think in the context of this file, it is more correct to just leave the old explanation.